lib/utils/effects.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+function knownEffects() {
+  return Object.keys(require("typed-redux-saga"));
+}
+
+function isEffectImport(value) {
+  return /^typed-redux-saga?/.test(value);
```

Comment:
Why is "typed-redux-sag" (without the final "a") a valid value here?

Reply:
Oh, good catch!